src/Symfony/Component/Form/OrderedFormConfigBuilderInterface.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form;
+
+/**
+ * @author GeLo <geloen.eric@gmail.com>
+ */
+interface OrderedFormConfigBuilderInterface extends OrderedFormConfigInterface
+{
+    /**
+     * Sets the form position.
+     *
+     *  * The position can be `null` to reflect the original forms order.
+     *
+     *  * The position can be `first` to place this form at the first position.
+     *    If many forms are defined as `first`, the original order between these forms is maintained.
+     *    Warning, `first` does not mean "very first" if there are many forms which are defined as `first`
+     *    or if you set up an other form `before` this form.
+     *
+     *  * The position can be `last` to place this form at the last position.
```

Comment:
What about using -1 here. Trying to get rid of "special" string values here. Not sure if that makes sense though.